redhat-director-scripts/docker/trilio-datamover/Dockerfile_tripleo_train_centos7

```diff
@@ -16,12 +16,16 @@ USER root
 ADD trilio.repo /etc/yum.repos.d/
 
 RUN yum clean all
-RUN yum install tvault-contego puppet-triliovault python-s3fuse-plugin-cent7 -y
+RUN yum install python3 -y
+RUN python3 -m venv /home/tvault/.s3venv
+RUN /home/tvault/.s3venv/bin/pip install --upgrade pip
+RUN /home/tvault/.s3venv/bin/pip install --extra-index-url "https://pypi.fury.io/triliodata-4-1/" s3fuse --upgrade --no-cache-dir
+RUN yum install tvault-contego puppet-triliovault -y
```

Comment:
Could you please move this step to 19th line and the newly added steps below that as puppet-triliovault can cause regression